flake8_pep3101.py

```diff
@@ -14,10 +13,7 @@ def __init__(self, tree, filename):
         self.tree = tree
 
     def run(self):
-        if self.filename == 'stdin':
-            lines = pycodestyle.stdin_get_value()
```

Comment:
@iham now I'm confused, in all other flake8 plugins the fix for handling stdin input was to use this approach rather that the self.tree as you are doing here, should it be changed everywhere as well? 🤔